src/System.Private.ServiceModel/tools/setupfiles/CleanupCertificates.cmd

```diff
@@ -0,0 +1,25 @@
+echo off
+setlocal
+
+pushd  %~dp0..\..\..\..\bin
+REM Locate any CoreRun.exe to be able to execute CertificateCleanup.exe
+REM Because CertificateCleanup itself is a test project, we are guaranteed
+REM it will contain CoreRun.exe even if no other tests have been built.
+for /f "delims=" %%A in ('where /F /R tests corerun.exe') do set "CoreRunVar=%%A"
+if '%CoreRunVar%' EQU '' (
+  Echo Could not cleanup certificiates because was unable to locate CoreRun.exe under %~dp0..\..\..\..\bin
+  goto done
+)
+
+for /f "delims=" %%A in ('where /F /R tests CertificateCleanup.exe') do set "CertCleanupVar=%%A"
+if '%CoreRunVar%' EQU '' (
+  Echo Could not cleanup certificiates because was unable to locate CertificateCleanup.exe under %~dp0..\..\..\..\bin
+  goto done
+)
+
+echo Cleaning up certificates using: %CoreRunVar% %CertCleanupVar%
+%CoreRunVar% %CertCleanupVar%
+
```

Comment:
Reviewers: this technique is a little unorthodox and comments are welcome.  We need to find where CoreRun.exe and CertificateCleanup.exe were placed by the build, but we don't have access to the MSBuild properties that determine their full path.  This CMD file just locates where they landed and executes them.  I will create a similar .sh script for Linux in a separate PR.